networking/multiple_networks/primary_networks/about-primary-nwt-nad.adoc

```diff
@@ -15,7 +15,7 @@ You can manage the life cycle of an additional network created by NAD with one o
 
 * By modifying the Cluster Network Operator (CNO) configuration. With this method, the CNO automatically creates and manages the `NetworkAttachmentDefinition` object. In addition to managing the object lifecycle, the CNO ensures that a DHCP is available for an additional network that uses a DHCP assigned IP address.
 
-* By applying a YAML manifest. With this method, you can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
+* By applying a YAML manifest. With this method, you can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the execution of multiple CNI plugins.
```

Comment:
@dougbtv PTAL

Reply:
I instructed Steven to change this wording because chaining has a specific technical connotation in terms of CNI, with Multus you're not actually chaining when you attach multiple interfaces. It'd be a chain on a single CNI ADD, more details [in the spec](https://github.com/containernetworking/cni/blob/main/SPEC.md#overview-1)

Like, if there's two interfaces, psuedo code

```
kind: net-attach-def
name: net-a
spec.config: { "type": "macvlan"}
```

And:

```
kind: net-attach-def
name: net-b
spec.config: { "type": "bridge"}
```

Those aren't actually chained CNI plugins. It's the execution of two separate CNI plugins with two separate CNI ADDs.

but you could have:

```
kind: net-attach-def
name: net-c
spec.config: { "plugins": [{type: "macvlan"},{type: "sysctl"}}
```

That's a chain. It's a technicality but I wanted to avoid using the terminology "chaining" as at the CNI API level it has another meaning.

Reply:
Maybe it can read better though, let's try:

instead of:

>  This approach allows for the execution of multiple CNI plugins.

try:

> This approach allows for the invocation of multiple CNI plugins in order to attach additional network interfaces in a pod.